Sources/SwiftDocC/Model/Rendering/Content/RenderContentMetadata.swift

```diff
@@ -20,6 +20,15 @@ public struct RenderContentMetadata: Equatable, Codable {
     public var abstract: [RenderInlineContent]?
     /// An optional identifier for the device frame that should wrap this element.
     public var deviceFrame: String?
+
+    /// Declare a public initializer, to allow clients linking to Swift DocC to
+    /// create RenderContentMetadata values.
```

Comment:
This comment isn't suitable as public documentation for this API. 

We could either:
- Remove this
   ```suggestion
   ```

- Replace it with a "create a new ..." comment without documenting the parameters:
   ```suggestion
       /// Creates a new metadata value for a content element.
   ```

- Replace it with a "create a new ..." comment and document the parameters based on their current property documentations:
   ```suggestion
       /// Creates a new metadata value for a content element.
       /// - Parameters:
       ///   - anchor: The named anchor of the content element.
       ///   - title: The customized title for the content element.
       ///   - abstract: The customized abstract for the content element.
       ///   - deviceFrame: The identifier for the device frame that should wrap the content element.
   ```